backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/utils/ApplicationHandler.kt

```diff
@@ -88,16 +89,16 @@ class ApplicationHandler(
     }
 
     fun isValidPreVerifiedApplication(): Boolean {
-        val isAlreadyVerifiedSet =
-            application.applicationDetails.blueCardEntitlement?.workAtOrganizationsEntitlement?.list?.any {
-                it.isAlreadyVerified == true
-            } ?: false
-        if (isAlreadyVerifiedSet) {
-            // check if api token is set and valid, if not throw unauthorized exception
-            // Will be done in #1790
-            throw UnauthorizedException()
+        val isAlreadyVerifiedList =
+            application.applicationDetails.blueCardEntitlement?.workAtOrganizationsEntitlement?.list?.map { it.isAlreadyVerified }
+                ?: emptyList()
+        return when {
+            isAlreadyVerifiedList.all { it == false || it == null } -> false
+            isAlreadyVerifiedList.all { it == true } -> {
+                throw UnauthorizedException()
```

Comment:
Since you wrote in the description to test with `alreadyVerified= true` to test if `VEREIN360` is set.
If i try this i end up here, since its currently "blocked"

Reply:
You are right, but as I wanted to avoid that at any point we have this enpoint unprotected on main, i added this exeption. You can either remove the exception by commenting it out and test or you just do not test it here, but do all the testing here: https://github.com/digitalfabrik/entitlementcard/pull/1811/